packages/sdks-tests/src/tests/blocks.spec.ts

```diff
@@ -287,4 +289,120 @@ test.describe('Blocks', () => {
       });
     }
   });
+
+  test.describe('Test ApiVersion', () => {
+    testOnlyOldReact('apiVersion in old react is no set', async ({ page }) => {
+      await page.goto('/api-version-default');
+
+      let x = 0;
+      await page.route(
+        /.*cdn\.builder\.io\/api\/v1\/content\/symbol.*/,
+        route => {
+          x++;
+          return route.fulfill({
+            status: 200,
+            body: x === 0 ? FIRST_SYMBOL_CONTENT : SECOND_SYMBOL_CONTENT,
+          });
+        },
+        { times: 2 }
+      );
+
+      await testSymbols(page);
+    });
+
+    testOnlyOldReact('apiVersion in old react is set to v1', async ({ page }) => {
+      await page.goto('/api-version-v1');
+
+      let x = 0;
+      await page.route(
+        /.*cdn\.builder\.io\/api\/v1\/content\/symbol.*/,
+        route => {
+          x++;
+          return route.fulfill({
+            status: 200,
+            body: x === 0 ? FIRST_SYMBOL_CONTENT : SECOND_SYMBOL_CONTENT,
+          });
+        },
+        { times: 2 }
+      );
+
+      await testSymbols(page);
+    });
+
+    testOnlyOldReact('apiVersion in old react is set to v3', async ({ page }) => {
+      await page.goto('/api-version-v3');
+
+      let x = 0;
+      await page.route(
+        /.*cdn\.builder\.io\/api\/v3\/content\/symbol.*/,
+        route => {
+          x++;
+          return route.fulfill({
+            status: 200,
+            body: x === 0 ? FIRST_SYMBOL_CONTENT : SECOND_SYMBOL_CONTENT,
+          });
+        },
+        { times: 2 }
+      );
+
+      await testSymbols(page);
+    });
+
+    testExcludeOldReact('apiVersion in new SDKs is not set', async ({ page }) => {
+      await page.goto('/api-version-default');
+
+      let x = 0;
+      await page.route(
+        /.*cdn\.builder\.io\/api\/v2\/content\/symbol.*/,
+        route => {
+          x++;
+          return route.fulfill({
+            status: 200,
+            body: x === 0 ? FIRST_SYMBOL_CONTENT : SECOND_SYMBOL_CONTENT,
+          });
+        },
+        { times: 2 }
+      );
+
+      await testSymbols(page);
+    });
+
+    testExcludeOldReact('apiVersion in new SDKs is set to v2', async ({ page }) => {
+      await page.goto('/api-version-v2');
+
+      let x = 0;
+      await page.route(
+        /.*cdn\.builder\.io\/api\/v2\/content\/symbol.*/,
+        route => {
+          x++;
+          return route.fulfill({
+            status: 200,
+            body: x === 0 ? FIRST_SYMBOL_CONTENT : SECOND_SYMBOL_CONTENT,
+          });
+        },
+        { times: 2 }
+      );
+
+      await testSymbols(page);
+    });
+
+    testExcludeOldReact('apiVersion in new SDKs is set to v3', async ({ page }) => {
```

Comment:
You can delete the other test and have this one run on all using `test()`

```suggestion
    test('apiVersion in SDKs is set to v3', async ({ page }) => {
```